.github/pull_request_template.md

```diff
@@ -6,12 +6,11 @@ Please provide a paragraph or two giving a summary of the change, including rele
 
 - [ ] I have reviewed my diff in github, line by line.
 - [ ] Every change is related to the PR description.
-- [ ] I have [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this pull request to the issue(s) that it resolves.
```

Comment:
This is less important and is rephrased to not imply that every PR must correspond to an issue.